### PR TITLE
Updates setup and adds CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by amoose**
### Completes https://github.com/fugacious/fugacious/issues/53

Adds bin/setup for quick app setup
Adds Procfile for use with foreman
Updates README's getting started and contributing
Adds CONTRIBUTING.md

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+We hope that you will consider contributing to Fugacious. You can contribute in many ways. For example, you might:
+
+* add documentation and "how-to" articles to the README or Wiki.
+* create new features or new integrations
+* improve and/or add new I18n translations
+* hack on Fugacious itself by fixing bugs you've found in the GitHub Issue tracker or adding new features to Fugacious.
```
Comment (jgrevich) [inline CONTRIBUTING.md]: the second part of the sentence seems redundant to the second bullet, "create new features or new integrations"

File: Procfile [hunk 2]
```diff
@@ -0,0 +1,2 @@
+web: bundle exec rails server
```
Comment (jgrevich) [inline Procfile]: You beat me to it :)

File: bin/setup [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+                                .so`.m:                                                             
+                               /y` -d-                                                              
+                               h..oo`                                                               
+                                -.                                                                  
+
+  ]
+
+
+  puts "\n== Installing dependencies =="
+  run "gem install bundler --conservative"
```
Comment (jgrevich) [inline bin/setup]: --no-ri --no-rdoc may be worth adding to speed things up, although those things perhaps are best left to each dev via their `.gemrc` config.

File: CONTRIBUTING.md (hunk 1 above)
Comment (amoose) [inline CONTRIBUTING.md]: it's true, I borrowed this from Devise

**Pull request merged**